src/tree/VirtualMachineTreeItem.ts

```diff
@@ -23,14 +23,16 @@ export class VirtualMachineTreeItem extends AzureTreeItem {
     }
 
     public get id(): string {
-        return nonNullProp(this.virtualMachine, 'id');
+        // https://github.com/microsoft/vscode-azurevirtualmachines/issues/70
+        return nonNullProp(this.virtualMachine, 'id').toLowerCase();
     }
 
     public get name(): string {
         return nonNullProp(this.virtualMachine, 'name');
     }
 
     public get resourceGroup(): string {
+        // https://github.com/microsoft/vscode-azurevirtualmachines/issues/70
         return `${getResourceGroupFromId(this.id).toLocaleLowerCase()}`;
```

Comment:
Should probably change this to `ToLowerCase()`. I don't think rg names can be non-unicode characters.